lib/CodeSnippitButton.php

```diff
@@ -14,15 +14,18 @@ function __construct( $cpt, $language ) {
 
 		// Set default programming language taxonomy terms
 		add_action( 'admin_init', array( $this, 'admin_init' ) );
+
+		add_action( 'wp_ajax_insert_snippet', array( $this, 'ajax_insert_snippet' ) );
 	}
 
 	public function button_script() {
 		wp_register_script( $this->script, DWSNIPPET_URL .'/lib/js/'. $this->script .'.js' , array( 'quicktags', 'wpdialogs' ), CodeSnippitInit::VERSION, true );
 		wp_localize_script( $this->script, 'codeSnippetCPT', array(
-			'buttons' => array( 'Cancel' => 'cancel', 'Insert Shortcode' => 'insert' ),
-			'button_img' => DWSNIPPET_URL .'lib/js/icon.png',
-			'button_name' => __( 'Add Snippet', 'code-snippet-cpt' ),
+			'buttons'      => array( 'Cancel' => 'cancel', 'Insert Shortcode' => 'insert' ),
```

Comment:
The button text needs to be localized, & you should swap the indexes with the values (so the slugs are the indexes).